src/OrbitQt/ProfilingTargetDialog.cpp

```diff
@@ -254,6 +256,13 @@ void ProfilingTargetDialog::SetupStadiaStates() {
   // STATE s_s_process_selected
   state_stadia_process_selected_.addTransition(this, &ProfilingTargetDialog::NoProcessSelected,
                                                &state_stadia_no_process_selected_);
+  QObject::connect(&state_stadia_process_selected_, &QState::entered, [this] {
```

Comment:
This should have `this` as its third argument and the lambda as the fourth. This will tell Qt that `this` needs to be alive, to execute this lambda. Probably that won't be a problem in the current state of the code, but it's a subtle thing. Someone might change the lifetime of `state_stadia_process_selected_` and then this might crash.

Reply:
I actually thought there wasn't a difference between the two versions. But you are right. 

(https://wiki.qt.io/New_Signal_Slot_Syntax)
> There is no automatic disconnection when the 'receiver' is destroyed because it's a functor with no QObject. However, since 5.2 there is an overload which adds a "context object". When that object is destroyed, the connection is broken (the context is also used for the thread affinity: the lambda will be called in the thread of the event loop of the object used as context).

I might go through other ui code and look for other times `this` was omitted. 

Thank you!

Reply:
Found one more in `ConnectToStadiaWidget.cpp`. done